subsys/bluetooth/services/fast_pair/fp_gatt_service.c

```diff
@@ -108,6 +195,54 @@ static int fp_gatt_rsp_notify(struct bt_conn *conn, const struct bt_gatt_attr *a
 	return err;
 }
 
+static int notify_personalized_name(struct bt_conn *conn)
+{
+	int err;
+	char pn[FP_STORAGE_PN_BUF_LEN];
+	size_t pn_len;
+	uint8_t nonce[FP_CRYPTO_ADDITIONAL_DATA_NONCE_LEN];
+	struct bt_gatt_attr *additional_data_attr;
```

Comment:
You can consider using `static` here and then you can use `bt_gatt_find_by_uuid` function only once under condition that `additional_data_attr` is NULL. It would save some time because `bt_gatt_find_by_uuid` needs to iterate whole attributes table and compares UUID's values.